examples/libxml2-ef709ce2.sh

```diff
@@ -19,8 +19,9 @@ cd obj-aflgo; CFLAGS="$ADDITIONAL" CXXFLAGS="$ADDITIONAL" ../configure --disable
 make clean; make -j4
 cat $TMP_DIR/BBnames.txt | rev | cut -d: -f2- | rev | sort | uniq > $TMP_DIR/BBnames2.txt && mv $TMP_DIR/BBnames2.txt $TMP_DIR/BBnames.txt
 cat $TMP_DIR/BBcalls.txt | sort | uniq > $TMP_DIR/BBcalls2.txt && mv $TMP_DIR/BBcalls2.txt $TMP_DIR/BBcalls.txt
-$AFLGO/distance/gen_distance_orig.sh $SUBJECT $TMP_DIR xmllint
+$AFLGO/distance/gen_distance_orig.sh $SUBJECT/obj-aflgo $TMP_DIR xmllint
```

Comment:
Why use obj-aflgo and not $PWD like in the other example scripts?

Reply:
In original `libxml2-ef709ce2.sh`:

### 1ST
https://github.com/aflgo/aflgo/blob/ac9246a4fe7c0bba15e36c994c456af14d89b698/scripts/fuzz/libxml2-ef709ce2.sh#L4
Now `SUBJECT` is `/path/to/libxml2_ef709ce2` and `TMP_DIR` is `/path/to/libxml2_ef709ce2/obj-aflgo/temp`.
### 2ND
https://github.com/aflgo/aflgo/blob/ac9246a4fe7c0bba15e36c994c456af14d89b698/scripts/fuzz/libxml2-ef709ce2.sh#L14-L15
Now all `*.bc` and other object files are located in `/path/to/libxml2_ef709ce2/obj-aflgo`.
### 3RD
https://github.com/aflgo/aflgo/blob/ac9246a4fe7c0bba15e36c994c456af14d89b698/scripts/fuzz/libxml2-ef709ce2.sh#L18
It equals `$AFLGO/scripts/genDistance.sh /path/to/libxml2_ef709ce2 /path/to/libxml2_ef709ce2/obj-aflgo/temp xmllint`.
This does work because `find` finds `xmllint.0.0.preopt.bc` downward to `obj-aflgo`.
But when `gen_distance_fast.py` faces all this, it would find nothing with `pathlib.Path("/path/to/libxml2_ef709ce2").glob("*.0.0.*.bc")`.

I fixed this strange behavior in 1e9c90c by adding `-maxdepth 1` for `find`.

To fit this, I just simply append `/obj-aflgo` to `$SUBJECT` here and remove `$SUBJECT` in other scripts.

Also, using `$PWD` instead and getting rid of `$SUBJECT` thoroughly may be a better choice...

BTW, why `$SUBJECT` was used in each of original fuzzing scripts? It looks like a dull copy-paste, and really makes me thinking if it has a mysterious intention...

Reply:
Cheers for this elaborate explanation!

> BTW, why $SUBJECT was used in each of original fuzzing scripts? It looks like a dull copy-paste, and really makes me thinking if it has a mysterious intention...

You are right. There is a lot of redundancy in these scripts.